docs/source/models.rst

```diff
@@ -87,6 +90,62 @@ signature
 input_example
   reference to an artifact with :ref:`input example <input-example>`.
 
+databricks_runtime
+    Databricks runtime version and type, if the model was saved in a Databricks notebook or job.
+
+
+
+
+Additional Logged Files
+^^^^^^^^^^^^^^^^^^^^^^^
+For environment recreation, two additional files are automatically logged whenever a model is saved. These files can then be used to reinstall dependencies through either ``conda`` or ``pip``.
+
+conda.yaml
+    When saving a model, MLflow provides the option to pass in a conda environment parameter that can contain dependencies used by the model. If no conda environment is provided, a default environment is created based on the flavor of the model. This conda environment is then saved in ``conda.yaml``.
+requirements.txt
+    The requirements file is created from the pip dependencies inside of the conda environment. Additional pip dependencies can be added to ``requirements.txt`` by including them as a pip dependency in a conda environment and logging the model with the environment. 
+
+The following shows an example of saving a model with a specified conda environment and the corresponding content of the generated ``conda.yaml`` and ``requirements.txt`` files.
+
+.. code-block:: py
+
+    conda_env = {
+        'channels': ['defaults', 'conda-forge'],
+        'dependencies': [
+            'python=3.8.8',
+            'pip'],
+        'pip': [
+            'mlflow',
+            'scikit-learn==0.23.2',
+            'cloudpickle==1.6.0'
+        ],
+        'name': 'mlflow-env'
+    }
+    mlflow.sklearn.save_model(model, "my_model", conda_env=conda_env)
+
+The written ``conda.yaml`` file:
+
+.. code-block:: text
+
+    channels:
+      - defaults
```

Comment:
Let's drop `defaults` here - I think we'll stop using this soon?

Reply:
We've dropped the `defaults` channel in https://github.com/mlflow/mlflow/pull/4393 :)